src/cirrus/docker.py

```diff
@@ -35,17 +37,15 @@ def __init__(self, cli_opts, config):
         self['login'] = config.get_param(
             'docker', 'docker_login_username', None
         ) is not None
-        self['tag'] = None
 
         if cli_opts.login:
             self['login'] = True
-        if hasattr(cli_opts, 'tag'):
-            if cli_opts.tag:
-                self['tag'] = cli_opts.tag
```

Comment:
Doesn't this remove the ability to override the tag via cli opts? Is this behavior intentional?